src/base/bittorrent/session.cpp

```diff
@@ -2557,7 +2558,15 @@ void Session::handleListenFailedAlert(libt::listen_failed_alert *p)
 void Session::handleExternalIPAlert(libt::external_ip_alert *p)
 {
     boost::system::error_code ec;
-    Logger::instance()->addMessage(tr("External IP: %1", "e.g. External IP: 192.168.0.1").arg(p->external_address.to_string(ec).c_str()), Log::INFO);
+    m_externalIp = p->external_address.to_string(ec);
+    Logger::instance()->addMessage(tr("External IP: %1", "e.g. External IP: 192.168.0.1").arg(m_externalIp.c_str()), Log::INFO);
```

Comment:
Minor change: 192.168.0.1 is a LAN IP, change it for a WAN IP e.g. 87.125.203.161